processors/osf/processor.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import logging
+import requests
+import datetime
+from .. import base
+logger = logging.getLogger(__name__)
+
+
+# Module API
+
+def process(conf, conn):
+
+    # Log started
+    logger.info('Started trials export')
+
+    # Export trials
+    # http://jamdb.readthedocs.io/en/latest/index.html
+    count = 0
+    limit = 100
+    token_ttl_seconds = 30*60  # api has limit 60*60 seconds
+    token_issued_time = datetime.datetime.now()
+    session = requests.Session()
+    for trials in _read_trial_groups(conn, limit=limit):
+
+        # Ensure authentificated
+        if (datetime.datetime.now() - token_issued_time).seconds > token_ttl_seconds:
+            del session.headers['Authorization']
+        if 'Authorization' not in session.headers:
+            url = '%s/auth' % conf['OSF_URL']
+            res = session.post(url, json={
+              'data': {
+                'type': 'users',
+                'attributes': {
+                  'provider': 'osf',
+                  'access_token': conf['OSF_KEY'],
+                }
+              }
+            })
+            # Check status
+            # 200 - ok
+            if res.status_code not in [200]:
+                logger.error('Can\'t authentificate')
+                exit(1)
+            token = res.json()['data']['attributes']['token']
+            session.headers.update({'Authorization': token})
+            logger.info('Succesefully authentificated')
+
+        # Ensure collection exists
+        url = '%s/namespaces/%s/collections'
+        url = url % (conf['OSF_URL'], conf['OSF_NAMESPACE'])
+        res = session.post(url, json={
+            'data': {
+                'id': 'trials',
+                'type': 'collections',
+                'attributes': {},
+            }
+        })
+        # Check status
+        # 201 - created
+        # 409 - conflict
+        if res.status_code == 201:
+            logger.info('Created collection "trials"')
+        elif res.status_code not in [201, 409]:
```

Comment:
We already know here that `res.status_code != 201` (because of the previous if), so you can simply do `res.status_code != 409`. Also, why 409 is OK? Does it mean they already have the trial? It'd be good to explain this in a comment.